### DISCUSSION TITLE
Is it normal to see only yml files?

### DISCUSSION THREAD

**Discussion by rlarudqls630** [photoprism/photoprism#4140] [Q&A]
after storing all the phtos and i checked back to see the photos
However, I do see some photos with black thumbnail, then i checked the server files and I only see yml files instead of jpeg or raw files.
Is it normal to see only yml files?

**Comment by lastzero** [comment]
You are probably checking the storage folder, which only contains additionally created files:
- https://docs.photoprism.app/getting-started/docker-compose/#photoprismstorage

Instead, you should check the originals path, which contains your original media files:
- https://docs.photoprism.app/getting-started/docker-compose/#photoprismoriginals

**Reply by rlarudqls630** [reply]
oh.. this is very confusing....
i am using truenas so i have no idea where did all my files went...

**Reply by lastzero** [reply]
From what I know, TrueNAS uses a third-party integration, so you'll need to consult their documentation or ask the maintainer.

**Reply by rlarudqls630** [reply]
honestly, I just need to download them

**Reply by lastzero** [reply]
That's possible from the UI:
- https://docs.photoprism.app/user-guide/organize/download/
